inc/class-s3-uploads-wp-cli-command.php

```diff
@@ -361,6 +361,26 @@ public function rm( $args, $args_assoc ) {
 		WP_CLI::success( sprintf( 'Successfully deleted %s', $prefix ) );
 	}
 
+	/**
+	 * Ensable the auto-rewriting of media links to S3
+	 */
+	public function enable( $args, $assoc_args ) {
+		if ( false === add_option( 's3_uploads_enabled', 'enabled', '', 'yes' ) ) {
```

Comment:
Functionally, yes. The point of attempting to use `add_option()` first is to explicitly set the autoload argument to `yes`. That's not an argument for `update_option()`. While `update_option()` will default to autoloading if an option does not exist _now_, there's no guarantee it will do so in the future and we want to be very clear that the autoloading of `s3_uploads_enabled` here is intentional and not an artifact of updating a non-existent option.